### PR TITLE
add index page

### LINKED ISSUE #3: first widget

### LINKED ISSUE #15: Rename landing.html to index.html

### PR DESCRIPTION
**Pull request by nada-abuzaid**
Relates #3 and #15

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        <img class="logo" src="images-index/logo.png" alt="Logo">
+        <div class="text">
+            <h2 class="header-title"> Watch Videos to Learn </h2>
+            <p class="header-paragraph">We have amazing videos that can help <br /> you understanding the concepts
+            </p>
+        </div>
+        <a class="next-btn" ref="landing.html ">Next</a>
+
+    </div>
+    <!-- /.main -->
```
Comment (yaqootturman) [inline index.html]: Delete comment, please!

**Review by yaqootturman — CHANGES_REQUESTED** [review]

**Review by IsraaHamdi — CHANGES_REQUESTED** [review]
add Semantic Elements

File: index.html (hunk 1 above)
Comment (nada-abuzaid) [reply index.html]: ok, done

**Review by yaqootturman — APPROVED** [review]
I suppose the tags will change because PR relates to naming of file, so keep this in your mind, please!

**Review by IsraaHamdi — APPROVED** [review]

**Pull request merged**